websites/R/R-a-d.io/presence.ts

```diff
@@ -0,0 +1,92 @@
+const presence = new Presence({
+		clientId: "1073136952905318401",
+	}),
+	// Get current player status
+	playStatus = (): string => {
+		let status: string;
+		const playButtonText: string =
+			document.querySelector("#stream-play").textContent;
+		if (playButtonText === "Play Stream") status = "Stopped";
+		else if (playButtonText === "Stop Stream") status = "Playing";
+		else status = playButtonText;
+		return status;
+	},
+	// Get current song playing
+	currentSong = (): string => {
+		return document.querySelector("#np").textContent;
+	};
+
+presence.on("UpdateData", async () => {
+	// Current path
```

Comment:
All the comments aren't needed, it is a given that this is the path given the variable name

Reply:
I see. I'll remove it when I wake up today. Thanks for the feedback 👍